full/src/test/java/apoc/uuid/UUIDClusterRoutingTest.java

```diff
@@ -134,6 +134,14 @@ public void testUuidInstallAllowedOnlyInSysLeaderMember() {
         final String query = "CALL apoc.uuid.install($label)";
         uuidInSysLeaderMemberCommon(query, SYS_NON_LEADER_ERROR, DEFAULT_DATABASE_NAME,
             (session, label) -> {
+                clusterSession.writeTransaction(tx -> tx.run(format("CREATE CONSTRAINT `%1$s` IF NOT EXISTS FOR (n:`%1$s`) REQUIRE n.uuid IS UNIQUE", label)));
```

Comment:
Why do we have the trailing `$s` in the constraint label now?

Reply:
I replaced `%s` with `%1$s` because I also use the latter as the index name into the `String.format`, to verify just below that it propagated correctly

Reply:
Yeah but aren't you creating the wrong contraint? You are also using it in the label of the `IF NOT EXISTS FOR`

Reply:
If I understood correctly, I don't think it's a problem if the index name is equal to the label one. It's just to simplify it.
Anyway, the `apoc.uuid.install` shouldn't work if there isn't a correct uuid constraint.